.gitattributes

```diff
@@ -1,8 +1,3 @@
 testing/data/**/* filter=lfs diff=lfs merge=lfs -text
 testing/baselines/**/* filter=lfs diff=lfs merge=lfs -text
 testing/data/DATA_LICENSES.md !filter !diff !merge text
-documentation/content/icon/**/* filter=lfs diff=lfs merge=lfs -text
-documentation/content/gallery/*.jpg filter=lfs diff=lfs merge=lfs -text
-documentation/content/gallery/*.png filter=lfs diff=lfs merge=lfs -text
-documentation/content/gallery/*.gif filter=lfs diff=lfs merge=lfs -text
-documentation/content/gallery/*.webm filter=lfs diff=lfs merge=lfs -text
```

Comment:
Please note that we cant store image/videos outside of LFS and LFS raw storage is just not fast enough when loading the website. Current solution of storing them on github as "upload" is fine, it is like storing the images on any image upload website. However, to make sure we do not loose these images at any point, I think I could store them in a secondary repository, eg: f3d-app/f3d-media. wdyt @Meakk ?

Reply:
That's a solution yes. Or store them in a private cloud somewhere.

Reply:
which is the same.